vscode/src/chat/chat-view/ChatController.ts

```diff
@@ -528,6 +530,9 @@ export class ChatController implements vscode.Disposable, vscode.WebviewViewProv
 
     private async getConfigForWebview(): Promise<ConfigurationSubsetForWebview & LocalEnv> {
         const { configuration, auth } = await currentResolvedConfig()
+        const experimentalPromptEditorEnabled = await firstValueFrom(
+            featureFlagProvider.evaluatedFeatureFlag(FeatureFlag.CodyExperimentalPromptEditor)
+        )
```

Comment:
```suggestion
        const experimentalPromptEditorEnabled = configuration.internalUnstable
```

If this is only for internal / dev to dogfood, maybe we can use the local editor settings instead of a feature flag? We have a (less-known) configuration used for enabling features for internal dogfood that we could re-use if it helps :)

Reply:
I think it would, but how does this work for Cody web? Would be good to dogfood there too.

Reply:
I'll merge this as is since we also want to test it in cody web.